### PR TITLE
Dotenv install

### PR DESCRIPTION
**Pull request by MarcosFierro**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export DEV_DATABASE=alpha-blog-dev
```
Comment (AlvaroSenorale) [inline .env]: you don't need to add `export` here.
```suggestion
DEV_DATABASE=alpha-blog-dev
```

File: config.ru [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 # This file is used by Rack-based servers to start the application.
+require 'dotenv'
```
Comment (AlvaroSenorale) [inline config.ru]: this was added automatically, right?

File: config/application.rb [hunk 3]
```diff
@@ -8,6 +8,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+Dotenv::Railtie.load
```
Comment (AlvaroSenorale) [inline config/application.rb]: and this?

File: config.ru (hunk 2 above)
Comment (MarcosFierro) [reply config.ru]: No, it is required to do Dotenv::Railtie.load so env variables get setted

File: .env (hunk 1 above)
Comment (MarcosFierro) [reply .env]: "You may also add export in front of each line so you can source the file in bash:"
I think it sets the variable in the bash file, i didn't checked it

Comment (AlvaroSenorale) [reply .env]: That means if you want to, but in your case you are not sourcing the file

File: config.ru (hunk 2 above)
Comment (AlvaroSenorale) [reply config.ru]: But I think you don't need to add it here because you're already adding it in application.rb . Right?

Comment (AlvaroSenorale) [reply config.ru]: we have to add  load the env in one place only

**Pull request merged**